blog/_posts/2016-10-07-perifericas-at-conferences.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Perifericas at conferences!
+layout: post
+created_at: Sunday Oct 16 2016
+permalink: blog/2016-10-16-perifericas-at-confereces
```

Comment:
the permalink, created_at, and filename should all show the same date. could you change that please? We might still publish it earlier depending on when if we get other posts in on time or not, but just in case we don't please change the filename to include `2016-10-16-`.